src/lib/utils.mjs

```diff
@@ -18,3 +18,31 @@ export const resolveFormat = (argv) => {
 
   return argv.format;
 };
+
+/**
+ * Standardizes the region of a database path.
+ *
+ * @param {string | undefined} databasePath - The database path to standardize.
+ * @returns {string | undefined} The standardized database path.
+ * @throws {TypeError} If the database path is not a string.
+ */
```

Comment:
if this also confirmed the rg was valid, it could save us a lot of distributed correctness checks. thoughts?